custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/r_predictor.py

```diff
@@ -181,4 +181,42 @@ def _rlist_to_dict(rlist):
         return ret
 
     def _transform(self, **kwargs):
-        raise DrumCommonException("Transform feature is not supported for R")
+        input_binary_data = kwargs.get(StructuredDtoKeys.BINARY_DATA)
+        target_binary_data = kwargs.get(StructuredDtoKeys.TARGET_BINARY_DATA)
+        mimetype = kwargs.get(StructuredDtoKeys.MIMETYPE)
+        with capture_R_traceback_if_errors(r_handler, logger):
+            predictions = r_handler.outer_transform(
+                binary_data=ro.rinterface.NULL
+                if input_binary_data is None
+                else ro.vectors.ByteVector(input_binary_data),
+                target_binary_data=ro.rinterface.NULL
+                if target_binary_data is None
+                else ro.vectors.ByteVector(target_binary_data),
+                mimetype=ro.rinterface.NULL if mimetype is None else mimetype,
+                model=self._model,
+            )
+
+        with localconverter(ro.default_converter + pandas2ri.converter):
+            py_data_object = ro.conversion.rpy2py(predictions)
```

Comment:
I assume this does something very important but I have no idea what. Maybe leave a comment?

Reply:
i think its straight forward - converts R type to py